src/models/backup.js

```diff
@@ -246,6 +246,11 @@ export default {
       const search = yield select(store => { return store.backup.search })
       yield put({ type: 'query', payload: { ...search } })
     },
+    *syncBackupVolume({
+      payload,
+    }, { call }) {
+      yield call(syncVolume, payload.name)
```

Comment:
Hi @mantissahz , @derekbit , there is no next UI change after call `syncVolume`. 

Do we need to update the backup volume list or do something else?

Reply:
Refresh sounds good. @mantissahz WDYT?

Reply:
Oh.. review the code again. 

After user clicks `Sync backup volume` option, longhorn backend should sync (has) the latest backup. So when user enter volume details page (will call [queryBackup](https://github.com/a110605/longhorn-ui/blob/b5eee34447136346676ff5ddf02408919deb7913/src/models/backup.js#L59)), they should auto see the latest backup result.

e.g. Click pvc-2de0a609-fb06-4ed0-aa0c-988630fe9f2f enter detail page.
<img width="1496" alt="image" src="https://github.com/longhorn/longhorn-ui/assets/5744158/fa7b0e33-dbde-4a72-b035-12f80244e9bd">

Reply:
@mantissahz , if there is period for backend to pull the backups from target. 
The successfully message should change to `Successfully trigger backup volume ${${backVolName}} synchronization` since the sync process has not done at the moment user click the button.